src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -215,6 +215,20 @@ class DefaultStorageKeyAllocator
     {
         return StorageKeyName::Formatted("f/%x/e/%x/sc/%x", fabric, endpoint, idx);
     }
+    // Stores the OperationalStateList for each endpoint and each cluster with operational state aliases cluster.
+    // Advantages: It will save the RAM memory if there are many operational states in list.Besides this attribute
+    // only need to be read in few times.
+    static StorageKeyName OperationalStateOpStateList(EndpointId endpoint, ClusterId clusterId)
+    {
+        return StorageKeyName::Formatted("g/ops/sl/%x/%" PRIx32, endpoint, clusterId);
```

Comment:
Stores the OperationalStateList / OperationalPhaseList for each endpoint and each cluster with operational state aliases cluster.
Advantages: It will save the RAM memory if there are many operational states in list.Besides this attribute                                                                      
only need to be read in few times.

Are they make sense?

Reply:
My point was that we should be leaving it up to the application how it wants to store this data: storage, .text, RAM, whatever.  We should not be forcing it to be stored as a blob in our KVS.